lighthouse-cli/test/smokehouse/oopif-expectations.js

```diff
@@ -27,5 +27,48 @@ module.exports = [
         },
       },
     },
+    artifacts: {
+      IFrameElements: [
+        {
+          id: 'oopif',
+          src: 'https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/',
+          clientRect: {
+            width: '>0',
+            height: '>0',
+          },
+          isPositionFixed: false,
```

Comment:
can we assert anything about the frame here or do we not get this info about cross origin frames?

Reply:
also worth noting that we picked this page because it has a nested youtube embed iframe in it, should that be expected to appear in this list?

or because it's an OOPIF we don't expect our artifact to capture it at all? 

i.e. what do we do about frames that are in the devtools protocl frame tree but we can't access from the dom due to origin concerns

Reply:
added an assertion that frame is undefined. Unfortunately it doesn't look like we are able to access frame information about OOPIF, found this old comment from @paulirish when I was looking into the issue: https://github.com/cyrus-and/chrome-remote-interface/issues/276#issuecomment-435553068.

I don't think there are any such frames, I think it's the same frames that we can't access that also don't show up in the frametree.